sunpy/net/tests/test_fido.py

```diff
@@ -124,6 +124,15 @@ def test_save_path_pathlib(tmpdir):
         assert f"eve{os.path.sep}0" in f
 
 
+@pytest.mark.remote_data
+def test_save_path_cwd(tmpdir):
+    qr = Fido.search(a.Instrument('EVE'), a.Time("2016/10/01", "2016/10/02"), a.Level(0))
+
+    # Test when path is ./ for current working directory
+    files = Fido.fetch(qr, path="./")
```

Comment:
```suggestion
    files = Fido.fetch(qr, path=tmpdir.strpath+"./")
```
I think?

Reply:
hmm will this work? this seems to be creating a path with just a ./ at the end. I'm a bit confused about `tmpdir` - is this just a temporary directory created for the test?

Also then how would you test it's in your current directory?

Reply:
Also silly question - is it currently passing or is are the tests just skipping online tests?

Reply:
Yeah `tmpdir` create a temporary directory created for the test. 

I don't actually think we need to use tmpdir here, it should be ok as it.

The windows CI build is failing, not sure why its missing from the checks at the bottom of the PR but it is there on the commit.

Reply:
We should change directory into the tmp dir before running fetch. Otherwise this will pollute whatever directory the tests are run in.

Reply:
what about this ? I think this might work ...

Reply:
Looks good to me.